app/api-v2/routes/item/{id}/metadata/{metadataKey}.js

```diff
@@ -0,0 +1,54 @@
+const { getMetadataResponse } = require('../../../../../util/appUtil')
+
+module.exports = function () {
+  const doc = {
+    GET: async function (req, res) {
+      return getMetadataResponse(req.params.id, req.params.metadataKey, res)
+    },
+  }
+
+  doc.GET.apiDoc = {
+    summary: 'Get metadata',
+    parameters: [
+      {
+        description: 'Id of the item',
+        in: 'path',
+        required: true,
+        name: 'id',
+        allowEmptyValue: true,
+      },
+      {
+        description: 'Metadata key of the item',
```

Comment:
I don't think this description is quite right. Maybe something more like `Key of metadata item to fetch`. That might mean changing the desc of the `id` to `Id of the item to get metadata for` or similar